CHANGELOG.md

```diff
@@ -4,6 +4,35 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Added
+- A new page to the docs labeled "Conventions" has been added.
+- The methods `Telescope.__add__` and `Telescope.__iadd__` have been added to allow
+`Telescope` objects with different antennas or feeds to be combined.
+- The methods `Telescope.reorder_antennas` and `Telescope.reorder_feeds` have been
+added to allow reordering of antenna metadata.
+- The methods `UVParameter.get_from_form` and `UVParameter.set_from_form`, which allows
+one to perform selection/setting operations based on the `UVParameter.form` attribute.
+- Handling for MeasurementSet calibration subtypes "T Jones" (non-pol-specific antenna
+gains) and "D Jones" (polarization leakages) has been added to `MSCal`.
+- The `UVBeam.x_orientation` parameter has been deprecated, superseded by two new
+required parameters: `UVBeam.feed_array` and `UVBeam.feed_angle`, which describe the
+polarization and orientation of the beam (the former of which was already present but
+only required for efield beams).
+- The `Telescope.x_orientation` parameter has been deprecated, superseded by two new
+parameters: `Telescope.feed_array` and `Telescope.feed_angle`, which describe the
+polarization and orientation of the detectors for a given antenna.
+- The `Telescope.mount_type` and `UVBeam.mount_type` parameters have been added, which
+describe the mount and optics of a given antenna (e.g., HERA/MWA are "fixed", VLA/ALMA
```

Comment:
MWA is set to "phased" in mwa_beam, not fixed. Maybe just take MWA out of this parenthetical?

Reply:
Changed to the right type in 0ede0cb9b4bbc4628136211eb13c25fded476c26.